spec

```diff
@@ -3500,7 +3500,6 @@ endseg
 beginseg
     name "ovl_En_Zl2"
     include "build/src/overlays/actors/ovl_En_Zl2/z_en_zl2.o"
-    include "build/data/overlays/actors/z_en_zl2.data.o"
     include "build/data/overlays/actors/z_en_zl2.reloc.o"
```

Comment:
with shiftablitiy now, we've started NON_MATCHING reloc for overlays that arent fully matching. there are some existing examples in the spec you can take a look at.
Otherwise, looks good to me